tests/istioUtils.sh

```diff
@@ -21,25 +21,25 @@ TESTS_DIR="${ROOT}/tests"
 LOGLEVEL=10
 
 function create_rule() {
-    ${ISTIOCLI} -v=${LOGLEVEL} -n ${NAMESPACE} create -f ${1} \
+    ${K8CLI} -v=${LOGLEVEL} -n ${NAMESPACE} create -f ${1} \
```

Comment:
Can we keep this as istioctl?

Reply:
Hey, can you unblock this? Without this change, no PR to pilot would pass.
I approved the use of kubectl in place of istioctl, so I can fix it back later, but let's first get this to pass.